pkg/envoy/rds/route_config_test.go

```diff
@@ -0,0 +1,908 @@
+package rds
```

Comment:
Was this file moved using `git mv`? Curious because this file diff doesn't appear the same as `route_config.go`:
![image](https://user-images.githubusercontent.com/21697719/188942219-102077b4-ce97-41d3-8be8-149bdb5f0c15.png)

Reply:
I moved it and removed the original `route` folder through VS code directly, but I also had to make changes to `route_config.go` because I moved a few of the methods there to `builder.go` and also needed to make some of the methods in `route_config.go` exportable to call in builder.

Reply:
Sure, but could you try to confirm if file history can be preserved using `git mv` versus removing the file directly on your IDE?

Reply:
Still not working with `git mv` not sure why it considers route_config_test a new file altogether but preserves version history for the others. If I recreate the /route folder and add route_config_test back there, git lists it as a modification, but if I use `git mv` and move it to /rds and delete the /route folder, then it thinks it's a new file and doesn't preserve the file history.

Reply:
Based on discussing this offline, it seems the file was moved after being edited, instead of being moved first. This could cause the file history to be unexpected. @nshankar13 will try to fix this.

Reply:
Tried resetting, moving the file with `git mv`, and then applying edits on top of the moved files. However, git still thinks `rds/builder_test.go` was previously `rds/route/route_config_test.go`, and that `rds/route_config_test.go` is now a new file.

Reply:
`pkg/envoy/rds/route/route_config_test.go → pkg/envoy/rds/builder_test.go`

Reply:
This might have to do with how `git` detects renames: https://chelseatroy.com/2020/05/09/question-how-does-git-detect-renames/